renderer/brave_content_renderer_client.cc

```diff
@@ -187,3 +188,14 @@ void BraveContentRendererClient::WillDestroyServiceWorkerContextOnWorkerThread(
   ChromeContentRendererClient::WillDestroyServiceWorkerContextOnWorkerThread(
       v8_context, service_worker_version_id, service_worker_scope, script_url);
 }
+
+std::unique_ptr<blink::URLLoaderThrottleProvider>
+BraveContentRendererClient::CreateURLLoaderThrottleProvider(
+    blink::URLLoaderThrottleProviderType provider_type) {
+  return std::make_unique<BraveURLLoaderThrottleProviderImpl>(
+      browser_interface_broker_.get(), provider_type, this);
+}
+
+bool BraveContentRendererClient::ReadIsTor() const {
```

Comment:
`IsTorProcess`

Reply:
fixed in 2bf1be60fb4efbe59956a565c4d4d65824cae9f1